src/features/rewards/box/style.ts

```diff
@@ -23,20 +23,14 @@ const colors: Record<Type, string> = {
 export const StyledCard = styled(Card as ComponentType<CardProps>)`
   margin-bottom: 28px;
   font-family: Poppins, sans-serif;
-  overflow: hidden;
 `
 
 export const StyledFlip = styled<StyleProps, 'div'>('div')`
-  display: flex;
-  width: 200%;
-  overflow: hidden;
+  display: block;
 `
 
 export const StyledContentWrapper = styled<StyleProps, 'div'>('div')`
-  display: flex;
-  flex-basis: 50%;
-  margin-left: ${p => p.open ? '0' : '-50%'};
-  flex-wrap: wrap;
+  display: ${p => p.open ? 'block' : 'none'};
```

Comment:
this block needs to have flex and flex-wrap, so that toggle is positioned correctly